### DISCUSSION TITLE
Multiple RSS feeds per community?

### DISCUSSION THREAD

**Discussion by jeffpooley** [knowledgefutures/pubpub#1676] [Ideas]
Every community has an RSS feed automatically generated. It would be useful to have the ability to generate additional feeds, per Collection—as an opt-in ‘enable feed’ toggle in the Collection Settings.

I suspect that communities would have various uses for this. The need that we have at [mediastudies.press](https://mediastudies.press) is to separate out announcement.news Pubs from book-chapters Pubs, by way of a Announcements Community.

**Comment by gabestein** [comment]
Hey Jeff, hope you had a nice thanksgiving! If I understand your needs, I think you can actually do this now by adding some variables to include/exclude collections in the RSS feed URL. We've never really documented it, and I'll take this as my cue to! But for now, there's some info about how to do it in [this PR](https://github.com/pubpub/pubpub/pull/1032).

Let me know if you have any questions about it. One note is the filter takes the collection slug, which in settings is called the "link."

**Reply by jeffpooley** [reply]
Thanks so much, Gabe! This is exactly what I was looking for. Is this the basic syntax  (if I wanted to generated a feed with just announcements (a “blog” Collection) and another feed that excluded announcements? 

https://hms.pubpub.org/rss.xml?collection=blog

https://hms.pubpub.org/rss.xml?collection=-blog

(This for the hms.mediastudies.press Community). When I test both, they don’t seem to be filtering.

**Reply by gabestein** [reply]
`collections` with the plural, but otherwise yes!

**Reply by jeffpooley** [reply]
This works perfectly—so excited about this!

One minor issue is that the Community footer menu, with its required “RSS” item, seems uneditable—both its name, and the feed itself. See screenshot:

<img width="1074" alt="rss" src="https://user-images.githubusercontent.com/6372329/143955908-5067aa04-9a35-4b23-b082-44bfea4b2d9b.png">

**Reply by gabestein** [reply]
Ah. We've always thought of that as a default that would be unwise to delete or modify, since it would be easy to break, which would then make it impossible for readers to discover feeds on the page. But perhaps we should change that.

**Reply by jeffpooley** [reply]
I totally understand the rationale. The default does, though, throw a wrench in using filtered feeds—though of course they can be surfaced elsewhere, like in the menus.

**Reply by gabestein** [reply]
Yeah, I can't really see why we couldn't allow it to be edited, to be honest. Some folks might want to use some kind of feed service, for instance.

I might be missing something, though. Beyond being messy UX, does it cause other issues?

**Reply by jeffpooley** [reply]
For us, the (minor) issue with the default RSS in the footer is just the inability to provide segregated feeds—so that a user would always have this generic “RSS” link that’s the full firehose, when (thanks to that great filtering feature) we’d prefer they see, instead, the two separate feeds, labeled. Even the ability to hide the default link would solve this tiny issue.
